### PR TITLE
started song add

### PR DESCRIPTION
**Pull request by Gr8-Dayne**

### CODE REVIEW

File: src/main/java/com/daylong/songr/AlbumSongs.java [hunk 1]
```diff
@@ -11,24 +11,52 @@
     long id;
 
     @ManyToOne
-    Album album;
+    Album songParent;
```
Comment (JamesDansie) [inline src/main/java/com/daylong/songr/AlbumSongs.java]: This needs to match the mapped by in the Album class.
```suggestion
    Album album;
```

File: src/main/java/com/daylong/songr/AlbumSongs.java [hunk 2]
```diff
@@ -11,24 +11,52 @@
...
+    private String songName;
+    private double songDuration;
+    private int trackNumber;
+
+    public AlbumSongs(Long id, String songName, double songDuration, int trackNumber, Album songParent){
+        this.id = id;
+        this.songName = songName;
+        this.songDuration = songDuration;
+        this.trackNumber = trackNumber;
+        this.songParent = songParent;
```
Comment (JamesDansie) [inline src/main/java/com/daylong/songr/AlbumSongs.java]: Get rid of id and songParent. Those are handled by Jpa. If you manually change them you will be sad.

Comment (Gr8-Dayne) [reply src/main/java/com/daylong/songr/AlbumSongs.java]: Good call.

Though I am used to being sad....

**Review by Gr8-Dayne — COMMENTED** [review]
I saw what you recommended (thank you) and i will be resubmitting in the near future.

**Pull request merged**